.github/workflows/backport-assistant.yml

```
@@ -29,9 +29,22 @@ jobs:
           BACKPORT_TARGET_TEMPLATE: "release/{{.target}}.x"
           GITHUB_TOKEN: ${{ secrets.ELEVATED_GITHUB_TOKEN }}
           ENABLE_VERSION_MANIFESTS: true
+  backport-ent:
+    if: github.event.pull_request.merged && contains(join(github.event.pull_request.labels.*.name), 'backport/ent')
+    runs-on: ubuntu-latest
+    container: hashicorpdev/backport-assistant:0.4.0
+    steps:
+      - name: Trigger backport for Enterprise
+        uses: peter-evans/repository-dispatch@ff45666b9427631e3450c54a1bcbee4d9ff4d7c0 # v3.0.0
+        with:
+          token: ${{ secrets.ELEVATED_GITHUB_TOKEN }}
+          repository: hashicorp/consul-enterprise
+          event-type: ent-backport
+          client-payload: ${{ toJson(github.event) }}
   handle-failure:
     needs:
       - backport
+      - backport-ent
     if: always() && needs.backport.result == 'failure'
```

I think you'll need to adjust this to something like
```suggestion
    if: always() && (needs.backport.result == 'failure' || needs.backport-ent.result == 'failure')
```

---

d'oh!